RussianFilter/sections/specific.txt

```diff
@@ -117,6 +117,7 @@ tproger.ru##div[id^="tproger_ads_widget-"]
 tproger.ru###after-post-widget-container
 ||torrenter.net/templates/newtorrent/img/bnr/
 ria.ru##.banner__content
+6.pogoda.by##.border_main
```

Comment:
This ad loaded from external script. Can be blocked using
`||1k.by/user/informers/topproductsforpogodaby-js-inline.php`